doc/source/ray-air/api/integrations.rst

```diff
@@ -0,0 +1,230 @@
+.. _air-ml-integrations:
+
+Ray AIR Integrations with ML Libraries
+======================================
+
+.. currentmodule:: ray
+
+PyTorch
+-------
+
+There are 2 recommended ways to train PyTorch models on a Ray cluster.
+
+.. note::
+
+    If you're training PyTorch models with PyTorch Lightning, see :ref:`below <air-pytorch-lightning>`
+    for the available PyTorch Lightning Ray AIR integrations.
+
+See the options |:one:| |:two:| below, along with the usage scenarios and API references for each:
+
+|:one:| Vanilla PyTorch with Ray Tune
```

Comment:
I think this could be a `tabs` section to compress things a bit (later sections, too)

Reply:
I tried this out and it doesn't let me add links as the tab headers -- do you think it's ok as is?